templates/default/logback.xml.erb

```diff
@@ -114,4 +114,7 @@
 
   <logger name="org.apache.cassandra" level="INFO"/>
   <logger name="com.thinkaurelius.thrift" level="ERROR"/>
+  <% node['cassandra']['logback']['override_loggers'].each do |logger, level| %>
+  <logger name="<%=logger%>" level="<%=level%>"/>
```

Comment:
Sorry to nitpick but can you please add spaces around `logger` and `level` here? Plus we should update the list of attributes in the README.